### PR TITLE
Don't run UTs, ITs when there are only docs, helm and console changes. 

### PR DESCRIPTION
**Pull request by abhishekagarwal87**

### CODE REVIEW

**Review by imply-elliott — APPROVED** [review]
👍

File: .github/workflows/unit-and-integration-tests-unified.yml [hunk 1]
```diff
@@ -16,10 +16,22 @@
...
+      - 'web-console/**'
     branches:
       - master
       - /^\d+\.\d+\.\d+(-\S*)?$/ # release branches
   pull_request:
+    paths-ignore:
+      - 'docs/**'
+      - 'examples/**'
+      - 'helm/**'
+      - 'integration-tests*/**'
```
Comment (tejaswini-imply) [inline .github/workflows/unit-and-integration-tests-unified.yml:33]: We would want to run ITs in case of changes in these paths, Isn't it?

Comment (abhishekagarwal87) [reply .github/workflows/unit-and-integration-tests-unified.yml]: Fixed.

Comment (tejaswini-imply) [reply .github/workflows/unit-and-integration-tests-unified.yml]: I guess we can also add extra `paths-ignore` on individual workflows, suppose `'integration-tests*/**'` in `unit-tests.yml`, but that might also not let ITs run as per the current flow. We could alter the flow to run ITs if unit tests are either successful or skipped. But it's complex and can be done as part of a separate PR.

File: .github/workflows/unit-and-integration-tests-unified.yml [hunk 2]
```diff
@@ -16,10 +16,20 @@
 name: Unit & Integration tests CI
 on:
   push:
+    paths-ignore:
+      - 'docs/**'
+      - 'examples/**'
```
Comment (rohangarg) [inline .github/workflows/unit-and-integration-tests-unified.yml:21]: I think the `examples` folder used for setting up hadoop for ITs - should we ignore changes in that or let it run the full CI? The changes in `examples` folder are also very in-frequent.

**Review by rohangarg — COMMENTED** [review]
LGTM % comment

File: .github/workflows/unit-and-integration-tests-unified.yml (hunk 2 above)
Comment (abhishekagarwal87) [reply .github/workflows/unit-and-integration-tests-unified.yml]: You are right. fixed that too.

**Review by rohangarg — APPROVED** [review]

**Pull request merged**